examples/README.md

```diff
@@ -76,3 +76,120 @@ AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联
       2020/08/10 22:54:14 received: hello, FISCO BCOS, I am multi broadcast publisher! 4
       2020/08/10 22:54:16 received: hello, FISCO BCOS, I am multi broadcast publisher! 5
     ```
+
+## 身份验证单播案例
+
+常规场景中，任何一个监听了某topic的接收者都能接受到发送者推送的消息。但在某些场景下，发送者只希望特定的接收者能接收到消息，不希望无关的接收者能任意的监听此topic。针对此类场景，FISCO BCOS 推出了 topic 认证功能。 认证功能是指对于特定的topic消息，只允许通过认证的接收者接收消息。详细请参考：[Topic认证功能](https://fisco-bcos-doc-qiubing.readthedocs.io/en/latest/docs/manual/amop_protocol.html#topic)
+
+-   启动 AMOP 消息订阅者：
+
+    ```shell
+    # go run examples/amop_auth/sub/subscriber.go [endpoint] [topic]
+    > go run examples/amop_auth/sub/subscriber.go 127.0.0.1:20201 hello
+    
+    Subscriber success
+    2020/08/27 15:59:33 received: Hi, FISCO BCOS! 0
+    2020/08/27 15:59:35 received: Hi, FISCO BCOS! 1
+    2020/08/27 15:59:37 received: Hi, FISCO BCOS! 2
+    2020/08/27 15:59:39 received: Hi, FISCO BCOS! 3
+    ```
+  
+
+-   运行 AMOP 消息发布者：
+
+    ```shell
+    # go run examples/amop_auth/unicast_pub/publisher.go [endpoint] [topic]
+    > go run examples/amop_auth/unicast_pub/publisher.go 127.0.0.1:20200 hello
+    
+    publish topic success
+    2020/08/27 15:59:33 publish message: Hi, FISCO BCOS! 0 
+    2020/08/27 15:59:35 publish message: Hi, FISCO BCOS! 1 
+    2020/08/27 15:59:37 publish message: Hi, FISCO BCOS! 2 
+    2020/08/27 15:59:39 publish message: Hi, FISCO BCOS! 3
+    ```
+
+## 身份验证多播案例
+
+同理，FISCO BCOS 支持带身份验证的消息多播功能
+
+-   启动 AMOP 消息订阅者：
+
+    ```shell
+    # go run examples/amop_auth/sub/subscriber.go [endpoint] [topic]
+    > go run examples/amop_auth/sub/subscriber.go 127.0.0.1:20201 hello
+    
+    Subscriber success
+    2020/08/27 16:02:39 received: Hi, FISCO BCOS! 1
+    2020/08/27 16:02:41 received: Hi, FISCO BCOS! 2
+    2020/08/27 16:02:43 received: Hi, FISCO BCOS! 3
+    2020/08/27 16:02:45 received: Hi, FISCO BCOS! 4
+    ```
+
+-   运行 AMOP 消息发布者：
+
+    ```shell
+    # go run examples/amop_auth/multicast_pub/publisher.go [endpoint] [topic]
+    > go run examples/amop_auth/multicast_pub/publisher.go 127.0.0.1:20200 hello
+    
+    publish topic success
+    2020/08/27 16:02:37 publish message: Hi, FISCO BCOS! 0 
+    2020/08/27 16:02:39 publish message: Hi, FISCO BCOS! 1 
+    2020/08/27 16:02:41 publish message: Hi, FISCO BCOS! 2 
+    2020/08/27 16:02:43 publish message: Hi, FISCO BCOS! 3 
+    2020/08/27 16:02:45 publish message: Hi, FISCO BCOS! 4
+    ```
+
+# 合约开发案例
+
+在利用SDK进行项目开发时，对智能合约进行操作需要利用go-sdk的abigen工具将Solidity智能合约转换为Go文件代码。整体上主要包含六个流程：
+
+-   准备需要编译的智能合约
+-   配置好相应版本的solc编译器
+-   构建go-sdk的合约编译工具abigen
+-   编译生成go文件
+-   准备建立ssl连接需要的证书
+-   使用生成的go文件进行合约部署、调用
+
+详细可参考：[合约开发样例](https://fisco-bcos-documentation.readthedocs.io/zh_CN/latest/docs/sdk/go_sdk/contractExamples.html)
+
+## 合约开发案例
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (142:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)